### ISSUE TITLE
Would it be possible to train the model on multiple GPUs?

### ISSUE THREAD

**Issue by UCASUSTC** [vt-vl-lab/iCAN#25]
Hi，
I would like to train the model on HICO-Det dataset.

I trained my model on a single TITAN X GPU for HICO-DET dataset. It took about five days for iteration 1800000 times.It takes too long.

would it be possible to train the model on multiple GPUs? what changes would I have to make?

and How big is the batch_size of your network?

Thanks!

**Comment by gaochen315** [comment]
Currently our code doesn't support multi-GPU training. However, you can try a larger learning rate, e.g. something between 0.01~0.05. In this case the training time will be reduced tremendously.

**Comment by UCASUSTC** [comment]
ok,thanks!
